tutorials/Tutorial9_DPR_training.ipynb

```diff
@@ -0,0 +1,637 @@
+{
```

Comment:
<p>Let's change the "EXECUTABLE VERSION: colab" to a proper colab icon that is linked. This will look way nicer on the website version of this tutorial ... </p><p>I'd suggest we do it in all tutorials. </p><p> </p><p>You see the instructions for doing it here: https://colab.research.google.com/github/googlecolab/colabtools/blob/master/notebooks/colab-github-demo.ipynb#scrollTo=8QAWNjizy_3O</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack/pull/708/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done